### PR TITLE
MAINT: refactor names of variables in `reference_db/tests/test_format.py`

### PR DESCRIPTION
**Pull request by Sann5**
In `reference_db/tests/test_format.py` there are many variables named `dirpath`. This PR changes all of those variables to `path_to_file` which is more appropriate since they all contain paths to files and not directories.

### CODE REVIEW

**Review by misialq — CHANGES_REQUESTED** [review]
Hey @Sann5, maybe you can rather rename those to something like `filepath` or simply `fp` - no need for all this verbosity 😅

**Comment by Sann5** [comment]
> Hey @Sann5, maybe you can rather rename those to something like `filepath` or simply `fp` - no need for all this verbosity 😅

@misialq done :)

File: q2_types/reference_db/tests/test_format.py [hunk 1]
```diff
@@ -46,35 +46,35 @@ def test_dmnd_dir_fmt_fails_bad_name(self):
...
-        fmt_obj = EggnogRefBinFileFmt(dirpath, mode='r')
+        fp = self.get_data_path('good_eggnog/eggnog.taxa.db')
+        fmt_obj = EggnogRefBinFileFmt(fp, mode='r')
 
         fmt_obj.validate()
 
     def test_eggnog_dir_fmt_all_files(self):
-        dirpath = self.get_data_path('good_eggnog')
-        fmt_obj = EggnogRefDirFmt(dirpath, mode='r')
+        fp = self.get_data_path('good_eggnog')
```
Comment (misialq) [inline q2_types/reference_db/tests/test_format.py]: This is a directory.

File: q2_types/reference_db/tests/test_format.py [hunk 2]
```diff
@@ -46,35 +46,35 @@ def test_dmnd_dir_fmt_fails_bad_name(self):
...
 
         self.assertEqual(
                 len([(relpath, obj) for relpath, obj
                      in fmt_obj.eggnog.iter_views(EggnogRefBinFileFmt)]),
                 3)
 
     def test_eggnog_dir_fmt_single_file(self):
-        dirpath = self.get_data_path('single_eggnog')
-        fmt_obj = EggnogRefDirFmt(dirpath, mode='r')
+        fp = self.get_data_path('single_eggnog')
```
Comment (misialq) [inline q2_types/reference_db/tests/test_format.py]: Aslo a directory.

File: q2_types/reference_db/tests/test_format.py [hunk 3]
```diff
@@ -84,26 +84,26 @@ def test_eggnog_dir_fmt_single_file(self):
         fmt_obj.validate()
 
     def test_eggnog_dir_fmt(self):
-        dirpath = self.get_data_path('good_eggnog')
-        fmt_obj = EggnogRefDirFmt(dirpath, mode='r')
+        fp = self.get_data_path('good_eggnog')
```
Comment (misialq) [inline q2_types/reference_db/tests/test_format.py]: Same here.

File: q2_types/reference_db/tests/test_format.py [hunk 4]
```diff
@@ -84,26 +84,26 @@ def test_eggnog_dir_fmt_single_file(self):
...
+        fmt_obj = EggnogRefDirFmt(fp, mode='r')
 
         fmt_obj.validate()
 
     def test_eggnog_sequence_taxa_dir_fmt(self):
-        dirpath = self.get_data_path('eggnog_seq_tax')
-        fmt_obj = EggnogProteinSequencesDirFmt(dirpath, mode='r')
+        fp = self.get_data_path('eggnog_seq_tax')
```
Comment (misialq) [inline q2_types/reference_db/tests/test_format.py]: Aaaaand here 😅

**Comment by Sann5** [comment]
@misialq shit sorry, here we go again 😅

**Review by misialq — APPROVED** [review]
All good, thanks! @lizgehret could you cast your eye over thins one too? Thanks! 🙏

**Review by lizgehret — APPROVED** [review]

**Comment by lizgehret** [comment]
lgtm!

**Pull request merged**